### ISSUE TITLE
Milestone 5 Task 1: Flask Tutorial

### ISSUE THREAD

**Issue by FLX-20** [FLX-20/Data-Science-Toolkits-and-Architectures-group-1#60]
Go through Flask Tutorials, such as this [Flask Tutorial](https://www.digitalocean.com/community/tutorials/how-to-build-and-deploy-a-flask-application-using-docker-on-ubuntu-18-04)
I recoomend to look also through other tutorial.
If you find a interesting one share it in the comments.

**Comment by Tjerk8389** [comment]
Should I document what we have learned from the tutorial in the branch called 'report-5-chapter-1'?

**Comment by FLX-20** [comment]
yes please write something about your biggest learnings, additional sources and issues you tackled during the tutorial.
Correct, write all this in the  'report-5-chapter-1' branch. Make sure you create the branch from the `main-report-5-branch`.

**Comment by Tjerk8389** [comment]
I completed the tutorial and wrote down my findings of the tutorial in short in the 'report-5-chapter-1' branch.

**Comment by FLX-20** [comment]
Perfect

Then Tim can also add his biggest learning and issues/challenges. 
Afterwards we can regard this issue as completed.
